### PR TITLE
Crates are mailed with manifests taped to the outside by paprka Merge Conflict resolution

### PR DESCRIPTION
**Pull request by astralenigma**
Resolves #7896 Merge conflicts
Adds a tear_manifest() to crates to remove copypasta.

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -146,7 +151,7 @@
 
 /obj/structure/closet/crate/secure/New()
 	..()
-	overlays.Cut()
+	//overlays.Cut()
```
Comment (Jordie0608) [inline code/game/objects/structures/crates_lockers/crates.dm]: This can go.

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -10,16 +10,21 @@
 	var/rigged = 0
 	var/sound_effect_open = 'sound/machines/click.ogg'
 	var/sound_effect_close = 'sound/machines/click.ogg'
-
+	var/obj/item/weapon/paper/manifest/manifest
+	
 /obj/structure/closet/crate/New()
 	..()
+	overlays.Cut()
```
Comment (Jordie0608) [inline code/game/objects/structures/crates_lockers/crates.dm]: This needs to be in update_icon()

**Comment by Cheridan** [comment]
:turtle:

**Comment by pudl** [comment]
this guy's history is really fishy

**Comment by Cheridan** [comment]
@pudl not as bad as your post history

**Comment by astralenigma** [comment]
Damn here I thought my story was solid, where did I make a mistake? So that I can improve.

File: code/game/objects/structures/crates_lockers/crates.dm (hunk 2 above)
Comment (astralenigma) [inline code/game/objects/structures/crates_lockers/crates.dm]: Should I also move the secure's crate addition of overlay to the update_icon()? or should I instead add the update_icon() to the new() before the addition of the overlay?

Comment (astralenigma) [inline code/game/objects/structures/crates_lockers/crates.dm]: Or should I override secure's update_icon()?

File: code/game/objects/structures/crates_lockers/largecrate.dm [hunk 3]
```diff
@@ -4,13 +4,38 @@
 	icon = 'icons/obj/crates.dmi'
 	icon_state = "densecrate"
 	density = 1
+	var/obj/item/weapon/paper/manifest/manifest
+
```
Comment (astralenigma) [inline code/game/objects/structures/crates_lockers/largecrate.dm]: The fact that largecrate is not a crate prevents the use of the tear_manifest() unless I drag tear_manifest() to structures but then you could tear manifests from every structure (maybe in the future cargo could order complete machines).

**Comment by xxalpha** [comment]
Vou-me rir bastante se tu fores mesmo o pimento.

**Comment by astralenigma** [comment]
Quem é o pimento?

**Comment by PKPenguin321** [comment]
"the pepper" is paprika who originally made this PR. paprika is banned from everything /tg/ because he ban evaded after getting banned for being rude.
when they say your history looks fishy they're saying it kinda looks like you're paprika ban evading due to your short history on git

**Comment by astralenigma** [comment]
Ok that explains why paprika was not cleaning his own mess then, but no I just got kinda tired of waiting for cheridan and the code wasn't complicated to solve the conflicts.

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 4]
```diff
@@ -146,7 +151,9 @@
 
 /obj/structure/closet/crate/secure/New()
 	..()
```
Comment (Jordie0608) [inline code/game/objects/structures/crates_lockers/crates.dm]: I should've mentioned I meant this entire statement can go; all it does is call parent.

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 5]
```diff
@@ -231,13 +250,12 @@
```
Comment (astralenigma) [inline code/game/objects/structures/crates_lockers/crates.dm]: The method above says that when he gets card he only locks the crate.

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 6]
```diff
@@ -287,15 +305,14 @@
 	if(!broken && !opened  && prob(50/severity))
 		if(!locked)
 			src.locked = 1
-			overlays.Cut()
-			overlays += redlight
+			update_icon()
```
Comment (astralenigma) [inline code/game/objects/structures/crates_lockers/crates.dm]: So a emp_act Locks an unlocked crate?

**Comment by astralenigma** [comment]
I wonder if I should remove paprka as author and put myself.

File: html/changelogs/Paprka-crades.yml.txt [hunk 7]
```diff
@@ -0,0 +1,6 @@
+author: Paprka
```
Comment (optimumtact) [inline html/changelogs/Paprka-crades.yml.txt]: Just set this to whatever you want if you want to change the change log author, i.e author: AstralEnigma

You cannot have two, but you could add something in changes -tweak: "Based on work by paprka"

File: code/game/objects/structures/crates_lockers/crates.dm (hunk 5 above)
Comment (optimumtact) [inline code/game/objects/structures/crates_lockers/crates.dm]: I believe the allowed function does the access checks for locking/unlocking

File: code/game/objects/structures/crates_lockers/crates.dm (hunk 6 above)
Comment (optimumtact) [inline code/game/objects/structures/crates_lockers/crates.dm]: Yes, strange, but I think it's okay to leave it as is.

File: html/changelogs/Paprka-crades.yml.txt (hunk 7 above)
Comment (astralenigma) [inline html/changelogs/Paprka-crades.yml.txt]: Honestly when I started it, it was mostly paprka's work now I'm not so sure due to all the lines I had to change to make the merge. I'm still thinking of making another change that will be moving my proc in crates to structures so I can also use it in the largecrate.

File: code/game/objects/structures/crates_lockers/crates.dm (hunk 5 above)
Comment (astralenigma) [inline code/game/objects/structures/crates_lockers/crates.dm]: I have tested it and it only locks I don't see what did I change that could make it happen like that so I think someone else made it like that either by mistake or on purpose, whichever reason will affect if I change it or not.

File: html/changelogs/Paprka-crades.yml.txt (hunk 7 above)
Comment (JJRcop) [inline html/changelogs/Paprka-crades.yml.txt]: You can have two..

**Pull request merged**